frontend/src/metabase/containers/NewItemMenu/NewItemMenu.unit.spec.tsx

```diff
@@ -133,4 +163,112 @@ describe("NewItemMenu", () => {
       expect(screen.queryByText("Action")).not.toBeInTheDocument();
     });
   });
+
+  describe("interactive embedding with `entity_types` (EMB-230)", () => {
```

Comment:
there are basically 3 cases, since `entity_types=` and `entity_types=model,table` are equivalent since it's the default value when `entity_types` isn't defined.